scripts/run_simulator.py

```diff
@@ -36,46 +48,26 @@ def get_webots_parameters() -> tuple[Path, Path]:
     if not world_file.exists():
         raise RuntimeError("World file not found.")
 
+    if not (SIM_BASE / "venv").exists():
+        print("Please run the setup.py script before running the simulator.")
+        raise RuntimeError
+
     # Check if Webots is in the PATH
     webots = which("webots")
 
     # Find the webots executable, if it is not in the PATH
     if webots is None:
-        if sys.platform == "darwin":
-            webots = "/Applications/Webots.app/Contents/MacOS/webots"
-        elif sys.platform == "win32":
-            possible_paths = [
-                "C:\\Program Files\\Webots\\msys64\\mingw64\\bin\\webotsw.exe",
-                expandvars("%LOCALAPPDATA%\\Programs\\Webots\\msys64\\mingw64\\bin\\webotsw.exe"),
-            ]
-            for path in possible_paths:
+        for system_filter, path in POSSIBLE_WEBOTS_PATHS:
+            if sys.platform.startswith(system_filter):
+                print(f"Checking {path}")
                 if Path(path).exists():
                     webots = path
```

Comment:
Silly question, why can't we do `return Path(webots), world_file` here?

Reply:
We'd still need an else clause to handle returning when L56 does find an executable, but essentially either would be fine.

Reply:
Ah ok, I see :)